test/cases/wasm/import-memory/env.js

```diff
@@ -0,0 +1 @@
+export const mem = new WebAssembly.Memory({ initial: 1 });
```

Comment:
This Memory object is actually not used! Webpack will automatically adds a new Memory object in the ImportObject.

In the future we will be able to use the user's provided one.